src/WormholeJoin.sol

```diff
@@ -40,7 +40,7 @@ interface TokenLike {
 }
 
 interface FeesLike {
-    function getFees(WormholeGUID calldata, uint256, int256) external view returns (uint256);
+    function getFees(WormholeGUID calldata, uint256, int256, uint256) external view returns (uint256);
```

Comment:
sidenote: shouldn't this be named `getFee`?

Reply:
Hmm yeah actually `getFee` is probably slightly better.

Reply:
will change that

Reply:
done!